src/%IPM/Lifecycle/Module.cls

```diff
@@ -38,6 +38,12 @@ Method %Activate(ByRef pParams) As %Status
 		
 		Set tSC = ..Configure(.pParams)
 		$$$ThrowOnError(tSC)
+
+		If '$ListFind(..PhaseList,"Package") {
+			// Code cannot be deployed if it is to be reexported and packaged.
+			Set tSC = ..MakeDeployed(.pParams)
```

Comment:
Over here should check for dev mode and only call this when not in dev mode for kitting purposes.
```
Set tDevMode = $Get(pParams("DeveloperMode"),..DeveloperMode)
```